src/Tickets/Commerce/Legacy_Compat.php

```diff
@@ -72,10 +72,8 @@ public function set_legacy_module_name( $name ) {
 	 */
 	public function maybe_load_currency_code_from_tribe_commerce( $value ) {
 
-		$currency = Currency::get_currency_code();
```

Comment:
if we remove line 75 here we won't be able to get Tickets Commerce currency codes